client/my-sites/checkout/checkout/checkout-container.jsx

```diff
@@ -24,7 +22,6 @@ class CheckoutContainer extends React.Component {
 	state = {
 		headerText: '',
 		subHeaderText: '',
```

Comment:
Do we need `subHeaderText`?(since it was introduced by the AB test PR)

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/35728/commits/4d27869d6849e927e495261f64f6d0667da649b8